draft-ietf-acme-acme.md

```diff
@@ -1801,7 +1802,7 @@ entity must both:
 
 {{security-considerations}} documents how the challenges defined in this
 document meet these requirements.  New challenges will need to document how they
-do.
+meet these requirements.
```

Comment:
I had a lot of trouble figuring out what it meant.

Perhaps "New challenges will need similar documentation"?

I don't think that an implied "this" work.